File: lihua-biz/lihua-monitor/src/main/java/com/lihua/monitor/model/CacheMonitor.java
package com.lihua.monitor.model;

import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class CacheMonitor {

    /**
     * 缓存key前缀
     */
    private String keyPrefix;

    /**
     * 缓存key 标签
     */
    private String label;

    /**
     * 缓存 Key
     */
    @NotNull(message = "缓存key不存在")
    private String key;

    /**
     * 缓存值
     */
    private String value;

    /**
     * 缓存过期时间（分钟）
     */
    private Long expireMinutes;

    public CacheMonitor(String keyPrefix, String label) {
        this.keyPrefix = keyPrefix;
        this.label = label;
    }

    public CacheMonitor() {
    }
}

File: lihua-vue/src/api/monitor/cache/type/cache-monitor.ts
export interface CacheMonitor {
    /**
     * 缓存键值前缀
     */
    keyPrefix: string
    /**
     * 缓存标签
     */
    label: string
    /**
     * 缓存值
     */
    value: string
    /**
     * 过期时间
     */
    expireMinutes: number
}
